src/app/main/component/events/services/events.service.ts

```diff
@@ -23,6 +23,10 @@ export class EventsService implements OnDestroy {
     return this.http.post<any>(`${this.backEnd}events/create`, formData);
   }
 
+  public editEvent(formData: FormData): Observable<any> {
```

Comment:
Observable<boolean>

Reply:
Observable<boolean>

Reply:
Hi, I got some issues with Github, this change is not related to this pr.